### PR TITLE
Adding picture element sample

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: picture-element/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>&lt;dialog&gt; Element Sample</title>
```
Comment (jeffposnick) [inline picture-element/index.html]: dialog -> picture

File: picture-element/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <link href='//fonts.googleapis.com/css?family=Roboto:300italic,400italic,700italic,400,300,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+  <body>
+    <h1>&lt;picture&gt; Element Sample</h1>
+    <p>Available in <a href="http://www.chromestatus.com/features/5910974510923776">Chrome 37+</a></p>
```
Comment (jeffposnick) [inline picture-element/index.html]: URL should be http://www.chromestatus.com/feature/5910974510923776 (no 's' in 'feature).

File: picture-element/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Picture Element Sample
+===
+See https://googlechrome.github.io/samples/picture-element for a live demo.
+
+Learn more at http://www.chromestatus.com/features/5910974510923776
```
Comment (jeffposnick) [inline picture-element/README.md]: URL should be http://www.chromestatus.com/feature/5910974510923776 (no 's' in 'feature).

**Comment by addyosmani** [comment]
:+1:

**Comment by pearlchen-old** [comment]
Hey @devnook, do you mind if I swap out your butterfly for a different image? I was working on docs for WPD and convinced an illustrator to draw me an adorable cat specifically for my &lt;picture&gt; example.

**Comment by devnook** [comment]
Sure, go for it. It's just an example image:)

On 19 August 2014 21:31, Pearl Chen notifications@github.com wrote:

> Hey @devnook https://github.com/devnook, do you mind if I swap out your
> butterfly for a different image? I was working on docs for WPD and
> convinced an illustrator to draw me an adorable cat specifically for my
> example.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/GoogleChrome/samples/pull/1#issuecomment-52694115.

## 

Chief Chaos Architect
@devnook.com

This email can contain funny or confidential information. If you received
this email by mistake, please delete all copies, shut down the computer and
leave the building.

**Comment by pearlchen-old** [comment]
Made some updates: https://github.com/GoogleChrome/samples/pull/8

**Pull request merged**